### PR TITLE
New File Added:Assignment5 and assignment6

### PR DESCRIPTION
**Pull request by vinod2406**

### CODE REVIEW

File: vinod/Bootstrap/ASSIN6/boostrap assi6.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                        <a class="nav-link" href="#">Professional Details</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Login</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+        <div>
+            <center><h1>job portal</h1></center>
```
Comment (Pranjali1999) [inline vinod/Bootstrap/ASSIN6/boostrap assi6.html]: center tag is out dated.

File: vinod/Bootstrap/ASSIN6/bootstrap personal.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+                  </label>
+                </div><br>
+                <h1>Academic details</h1>
+                <label for="Education"><b>Education</b></label>
+                &emsp;&emsp;&emsp;<input type="text" placeholder="education" name="name" id="education" required /><br /><br />
+               <label for="University"><b>University</b></label>
+                &emsp;&emsp;&emsp;<input type="text" placeholder="university" name="name" id="university" required /><br /><br />
+                <div class="container">
+                  <div class="form-check-inline">
+                    <label class="form-check-label" >
```
Comment (Pranjali1999) [inline vinod/Bootstrap/ASSIN6/bootstrap personal.html]: Where is closing of this label.

File: vinod/Bootstrap/ASSIN6/bootstrap personal.html [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+                    <label class="checkbox-inline">
+                      <input type="checkbox" value="">HTML
+                    </label>
+                    <label class="checkbox-inline">
+                      <input type="checkbox" value="">JAVA
+                    </label>
+                    <label class="checkbox-inline">
+                      <input type="checkbox" value="">JAVA SCRIPT
+                    </label> 
+                    </div><br>
```
Comment (Pranjali1999) [inline vinod/Bootstrap/ASSIN6/bootstrap personal.html]: Give indentation properly.

**Review by Pranjali1999 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
make changes asap

**Review by akash19972306 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**